src/librustc_apfloat/ppc.rs

```diff
@@ -20,7 +20,7 @@ use std::ops::Neg;
 pub struct DoubleFloat<F>(F, F);
 pub type DoubleDouble = DoubleFloat<ieee::Double>;
 
-// These are legacy semantics for the Fallback, inaccrurate implementation of
+// These are legacy semantics for the Fallback, inaccurate implementation of
```

Comment:
[Please also submit a patch to LLVM](https://github.com/llvm-mirror/llvm/blob/f8dd899c6c05e91652a92f7f7e0d61255142780f/lib/Support/APFloat.cpp#L89)